source/openpower/keys.rst

```diff
@@ -0,0 +1,106 @@
+.. _keys:
+
+SSH Key Behavior on OpenPOWER
+=============================
+
+These are findings of how SSH Keys behave in various conditions.
+  
+Changing a Key From Inside an Instance
+--------------------------------------
+
+When we create our OpenPOWER OpenStack instance, we set up a key pair to create the machine with.
+
+.. note::
+
+  For the purposes of this documentation, this key pair will be known as the **Original Key Pair**.
+
+To allow for multiple SSH Keys to access this instance:
+  We need to add the contents of ``<our_key>.pub`` to ``~/.ssh/authorized_keys`` from the machine with the corresponding keyfile.
+  
+  To do so, we need to be certain we can ``ssh`` into our instance with:: 
```

Comment:
should I leave the part about replacing the old key or just go through the steps to use ssh-copy-id?